projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -11,37 +11,51 @@ import { store as editorStore } from '@wordpress/editor';
 import { createInterpolateElement, useEffect, useState } from '@wordpress/element';
 import { __, _n, sprintf } from '@wordpress/i18n';
 import InspectorNotice from '../../shared/components/inspector-notice';
-import { getSubscriberCounts } from './api';
 import './panel.scss';
+import { getSubscriberCounts } from './api';
 import { META_NAME_FOR_POST_LEVEL_ACCESS_SETTINGS } from './constants';
 import { NewsletterAccess, accessOptions, MisconfigurationWarning } from './settings';
 import { isNewsletterFeatureEnabled } from './utils';
 
-function AccessLevelSelectorPanel( { setPostMeta, accessLevel } ) {
+function AccessLevelSelectorPanel( {
+	setPostMeta,
+	accessLevel,
+	followerCount,
+	subscribersCount,
+	paidSubscribersCount,
+} ) {
 	if ( ! isNewsletterFeatureEnabled() ) {
 		return null;
 	}
 
 	return (
 		<PluginDocumentSettingPanel title={ __( 'Newsletter', 'jetpack' ) }>
-			<NewsletterAccess setPostMeta={ setPostMeta } accessLevel={ accessLevel } />
+			<NewsletterAccess
+				setPostMeta={ setPostMeta }
+				accessLevel={ accessLevel }
+				followerCount={ followerCount }
+				subscribersCount={ subscribersCount }
+				paidSubscribersCount={ paidSubscribersCount }
+			/>
 		</PluginDocumentSettingPanel>
 	);
 }
 
 export default function SubscribePanels() {
-	const [ subscriberCount, setSubscriberCount ] = useState( null );
+	const [ subscribersCount, setSubscribersCount ] = useState( null );
```

Comment:
Are we just renaming variables for fun?

Reply:
I think it was a typo. `subscribersCount` is the "count of subscriberS". So it should be plural I think?....

Reply:
Also it was spelled like that somewhere else...

Reply:
But then you have `followerCount` below :D

I don't think that is that important, but this change makes a ton of changes in many lines and that becomes harder to review and understand what the changes are important and which less so.

I'm fine either way!